packages/babel-traverse/src/scope/index.js

```diff
@@ -596,6 +596,8 @@ export default class Scope {
     } else if (t.isClassProperty(node)) {
       if (node.computed && !this.isPure(node.key, constantsOnly)) return false;
       return this.isPure(node.value, constantsOnly);
+    } else if (t.isUnaryExpression(node)) {
+      return this.isPure(node.argument);
```

Comment:
Are you supposed to pass the 2nd param?

`return this.isPure(node.argument, constantsOnly);`